1.0/en/0x10-C02-User-Input-Validation.md

```diff
@@ -62,6 +62,44 @@ Even syntactically valid prompts may request disallowed content (illicit instruc
 
 ---
 
+## C2.5 Input Rate Limiting & Abuse Prevention
+
+Prevent abuse, resource exhaustion, and automated attacks by limiting input rates and detecting anomalous usage patterns.
+
+| # | Description | Level | Role |
+|:--------:|---------------------------------------------------------------------------------------------------------------------|:---:|:---:|
+| **2.5.1** | **Verify that** per-user, per-IP, and per-API-key rate limits are enforced for all input endpoints. | 1 | D/V |
+| **2.5.2** | **Verify that** burst and sustained rate limits are tuned to prevent DoS and brute-force attacks. | 2 | D/V |
+| **2.5.3** | **Verify that** anomalous usage patterns (e.g., rapid-fire requests, input flooding) trigger automated blocks or escalations. | 2 | D/V |
+| **2.5.4** | **Verify that** abuse prevention logs are retained and reviewed for emerging attack patterns. | 3 | V |
+
+---
+
+## C2.6 Multi-Modal Input Validation
+
+Ensure robust validation for non-textual inputs (images, audio, files) to prevent injection, evasion, or resource abuse.
+
+| # | Description | Level | Role |
+|:--------:|---------------------------------------------------------------------------------------------------------------------|:---:|:---:|
+| **2.6.1** | **Verify that** all non-text inputs (images, audio, files) are validated for type, size, and format before processing. | 1 | D |
+| **2.6.2** | **Verify that** files are scanned for malware and steganographic payloads before ingestion. | 2 | D/V |
+| **2.6.3** | **Verify that** image/audio inputs are checked for adversarial perturbations or known attack patterns. | 2 | D/V |
+| **2.6.4** | **Verify that** multi-modal input validation failures are logged and trigger alerts for investigation. | 3 | V |
+
+---
+
+## C2.7 Input Provenance & Attribution
```

Comment:
Currently we have a separate category for [logging & abuse detection](https://github.com/OWASP/AISVS/blob/main/1.0/en/0x10-C12-Monitoring-and-Logging.md). I think it would be better to remove these from here, and then you can check for any suggestions to expand / revise the current content.